package.json

```diff
@@ -52,31 +52,36 @@
   "devDependencies": {
     "@commitlint/cli": "11.0.0",
```

Comment:
These two might better also be updated the same way as:
https://github.com/mattrglobal/bbs-signatures/pull/131/files

Also husky might also need to be updated to 8.0